app/src/main/java/fr/free/nrw/commons/customselector/helper/OnSwipeTouchListener.kt

```diff
@@ -16,7 +16,7 @@ open class OnSwipeTouchListener(context: Context?) : View.OnTouchListener {
     private val SWIPE_THRESHOLD_WIDTH = (getScreenResolution(context!!)).first / 3
     private val SWIPE_VELOCITY_THRESHOLD = 1000
 
-    override fun onTouch(view: View?, motionEvent: MotionEvent?): Boolean {
+    override fun onTouch(view: View?, motionEvent: MotionEvent): Boolean {
```

Comment:
Changed parameter type from motionEvent: MotionEvent? to MotionEvent.

Fixes error: `Type mismatch: inferred type is MotionEvent? but MotionEvent was expected`